defaults/main.yml

```
@@ -50,15 +50,15 @@ zsh_antigen_bundles:
   - zsh_reload
 
   # on zsh 4.3.10 autosuggestions breaks multiline commands with \
-  - { name: zsh-users/zsh-autosuggestions, when: "{{ zsh_version | default(false) | version_compare('5.0', '>=') }}" }
+  - { name: zsh-users/zsh-autosuggestions, when: "{{ zsh_version|default(false) | is version('5.0', '>=') }}" }
```

I think this should be 
```suggestion
  - { name: zsh-users/zsh-autosuggestions, when: "{{ zsh_version|default(false) is version('5.0', '>=') }}" }
```

---

Yes, my bad !! I forgot to remove this pipe